nf_core/modules/module_utils.py

```diff
@@ -57,7 +57,7 @@ def get_module_git_log(module_name, modules_repo=None, per_page=30, page_nbr=1,
     if modules_repo is None:
         modules_repo = ModulesRepo()
     api_url = f"https://api.github.com/repos/{modules_repo.name}/commits"
-    api_url += f"?sha{modules_repo.branch}"
+    api_url += f"?sha={modules_repo.branch}"
```

Comment:
:open_mouth:

Reply:
🧙🏻

Reply:
That one character required a lot of `print` statements to find.. 😆